jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -167,7 +160,8 @@
         <applicationService serviceImplementation="software.aws.toolkits.jetbrains.core.AwsSdkClient"/>
         <applicationService serviceInterface="software.aws.toolkits.jetbrains.services.telemetry.TelemetryService"
                             serviceImplementation="software.aws.toolkits.jetbrains.services.telemetry.DefaultTelemetryService"
-                            testServiceImplementation="software.aws.toolkits.jetbrains.services.telemetry.MockTelemetryService" />
+                            testServiceImplementation="software.aws.toolkits.jetbrains.services.telemetry.MockTelemetryService"
+                            preload="true"/>
```

Comment:
preload is new to me, what does it do?

Reply:
Does it exist in 192?

Reply:
Agh... it is 193. Reverted back. application components should be dropped in future releases due to performance issues. It can be rather changed to application listener, but I would rather make a separate commit for this.

Reply:
Agree that we should avoid components.